.github/workflows/publish-docs.yml

```
@@ -54,54 +47,32 @@ jobs:
-        run: |
-          mkdocs build
+        run: mkdocs build --strict
+      - name: upload build artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: mkdocs-site
+          path: docs/site
 
-      - name: Install Vercel CLI
-        run: npm install -g vercel
+deploy-docs:
```

actionlint (1.7.4)

unexpected key "deploy-docs" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)